var/spack/repos/builtin/packages/ucx/package.py

```diff
@@ -59,6 +61,15 @@ class Ucx(AutotoolsPackage, CudaPackage):
     conflicts('+gdrcopy', when='~cuda',
               msg='gdrcopy currently requires cuda support')
 
+    @property
+    def configure_abs_path(self):
+        # UCX docs recommend using configure-release
+        configure_abs_path = os.path.join(
+            os.path.abspath(self.configure_directory),
+            'contrib/configure-release'
+        )
+        return configure_abs_path
```

Comment:
I think this is equivalent:
```suggestion
    configure_abs_path = 'contrib/configure-release'
```